lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -567,177 +568,182 @@ def write(chunk)
         log_name = "projects/#{@project_id}/logs/#{log_name(
           tag, group_level_resource)}"
 
-        # Does the actual write to the cloud logging api.
+        # Pile up the requests in the REST format. The gRPC path will decompose
+        # them later.
+        requests_to_send << Google::Apis::LoggingV2::WriteLogEntriesRequest.new(
+          entries: entries,
+          log_name: log_name,
+          resource: group_level_resource,
+          labels: group_level_common_labels,
+          partial_success: @partial_success)
+      end
+      requests_to_send.each do |request|
         client = api_client
-        if @use_grpc
-          begin
-            labels_utf8_pairs = group_level_common_labels.map do |k, v|
-              [k.encode('utf-8'), convert_to_utf8(v)]
-            end
+        # Does the actual write to the cloud logging api.
```

Comment:
Nit: `Cloud Logging API`.

Reply:
Renamed it to `Stackdriver Logging API` since that's how we refer to it in [external docs](https://cloud.google.com/logging/docs/reference/v2/rest/).

Reply:
:+1:

Reply:
😃